alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportGetFindingsAction.kt

```diff
@@ -114,60 +120,56 @@ class TransportGetFindingsSearchAction @Inject constructor(
         searchSourceBuilder.query(queryBuilder)
 
         client.threadPool().threadContext.stashContext().use {
-            search(searchSourceBuilder, actionListener)
+            scope.launch {
+                search(searchSourceBuilder, actionListener)
```

Comment:
maybe don't pass `actionListener` here and handle exception and `onResponse()` right here.

Reply:
done